sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -839,6 +865,34 @@ export class ServiceBusAtomManagementClient extends ServiceClient {
     if (isUpdate) {
       webResource.headers.set("If-Match", "*");
     }
+
+    const queueOrSubscriptionFields = entityFields as
+      | InternalQueueOptions
+      | InternalSubscriptionOptions;
+    if (
+      queueOrSubscriptionFields.ForwardTo ||
+      queueOrSubscriptionFields.ForwardDeadLetteredMessagesTo
+    ) {
+      const token = (await this.sasTokenProvider.getToken(this.endpoint)).token;
+      const urlPrefix = `${this.endpointWithProtocol}`;
```

Comment:
We don't need this constant anymore. `this.endpointWithProtocol` can be used directly instead

Reply:
Removed